docs/cep/tutorials/create-stream-app.md

```diff
@@ -286,6 +286,73 @@ The stream worker will use the Macrometa collections with the default query para
 </tbody>
 </table>
 
+
+## Source, Sink, Table and Window: Understanding with the help of a Stream Worker example
+
+
+In the application below, a Macrometa document based collection plays role of `source` to our SW IntrusionDetectionSystem.
+Consider InboundTrafficData as an audit collection where client IPs are added as they make requests to the server.
+As a new record is added, an event is emitted by collection streams associated with this collection. This event is consumed by the `source` InboundTraffic.
+
+Now it is quite possible that your application, `IntrusionDetectionSystem` may get burst of events having the same IP, and you need to gauge  
+this traffic in some way. Maybe 50 requests in a couple of seconds from the same IP can be considered suspicious traffic.
+Windows store events as and when they arrive and automatically expire/clean them based on the given window constraint.
+
+In our IntrusionDetectionSystem application, we have associated a `sliding window` of 2 seconds on incoming events. The window will hold all the events that appear
+within 2 seconds. Further Stream QL is grouping all the events by IP and if the count on any of these groups is more than 50, i.e., more than 50 events occurred
+from the same IP within 2 seconds, we can assume this is suspicious traffic. 
+
+
+In addition to the window, we have reference data for our application, i.e., `Table`.  A table is a stored collection of events, and its schema is defined via the
```

Comment:
```suggestion
In addition to the window, we have a `Table` which serves reference data for our application.  A table is a stored collection of events, and its schema is defined via the table definition. Unlike `stream`, `table` is stateful and supports interactive queries for the state of the stored events. In our case `SuspiciousIPTable` stores pre-defined suspicious IPs which we are using to verify if it matches IPs in suspicious traffic.
```

Is it correct to say that a table "serves" reference data?  Or should we rephrase this?